### ISSUE TITLE
Add a warning message if properties in `config.properties` are incorrect.

### ISSUE THREAD

**Issue by Livila** [FeKozma/Code-check#28]
Since all keys and values for the properties should be in `config.properties`, we should add a warning logging message if it's incorrect. Currently we set a value a value that should be working if the configuration is incorrect (#21), however we should also give a warning message about it.

`config.properties` should contain all default configuration keys and values.
One can use the `local.properties` file to change properties locally.

Basically what needs to be done is to add a warning message if a property's key doesn't exist, or if the value is incorrect. #21

**Comment by Livila** [comment]
Related comment: https://github.com/FeKozma/Code-check/pull/21#discussion_r1508060078

**Comment by Livila** [comment]
Already fixed, or no need to do this.
